optaplanner-examples/src/main/java/org/optaplanner/examples/machinereassignment/optional/DeleteMachineEasierPFC.java

```diff
@@ -0,0 +1,54 @@
+package org.optaplanner.examples.machinereassignment.optional;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import org.optaplanner.core.api.solver.change.ProblemChange;
+import org.optaplanner.core.api.solver.change.ProblemChangeDirector;
+import org.optaplanner.examples.machinereassignment.domain.MachineReassignment;
+import org.optaplanner.examples.machinereassignment.domain.MrMachine;
+import org.optaplanner.examples.machinereassignment.domain.MrMachineCapacity;
+import org.optaplanner.examples.machinereassignment.domain.MrProcessAssignment;
+
+// TODO: Delete. This class is just for illustrating the PFC impl.
+public class DeleteMachineEasierPFC implements ProblemChange<MachineReassignment> {
+
+    private final MrMachine machine;
+
+    public DeleteMachineEasierPFC(MrMachine machine) {
+        this.machine = machine;
+    }
+
+    @Override
+    public void doChange(MachineReassignment workingSolution, ProblemChangeDirector problemChangeDirector) {
```

Comment:
The working solution is available as a parameter of the method, as it's usually the first thing every PFC retrieves.